src/mitol/google_sheets_refunds/api.py

```diff
@@ -125,7 +125,11 @@ def _int_filter_row(self, row_tuple):
         """Performs row filtering according to the rules noted in filter_ignored_rows"""
         try:
             parsed_data = RefundRequestRow.parse_raw_data(*row_tuple)
-            return not parsed_data.skip_row
+            return not (
+                parsed_data.skip_row
+                or parsed_data.refund_complete_date is not None
+                or len(parsed_data.errors) > 0
```

Comment:
Are you also skipping rows that had errors as well? I think we don't want to skip those on subsequent runs, since we want to get the processed correctly.

Reply:
Good call; pulled that in https://github.com/mitodl/ol-django/pull/99/commits/451875e33d3e5b464c693bb461896027346fe9ab .